app/experimenter/nimbus-ui/src/App.tsx

```diff
@@ -0,0 +1,17 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+
+const App = () => {
+  return(
+    <div data-testid="app"><section><p>
```

Comment:
Aaand I'll make sure Prettier is ran, I purposely did this weird spacing to myself to make sure I wouldn't forget, heh.